packages/stream-api/src/streamApi.ts

```diff
@@ -92,6 +92,17 @@ export class StreamApi {
     );
   }
 
+  public createSceneAlteration(
+    request: vertexvis.protobuf.stream.ICreateSceneAlterationRequest
+  ): Promise<vertexvis.protobuf.stream.IStreamResponse> {
```

Comment:
Pattern here is to include a `withResponse` as the last argument that defaults to `true`.